app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -251,8 +247,99 @@ class PatientComponent extends Component {
     this.setState({ gender: event.target.value });
   }
 
+  /**
+   * This method handles the validation of the user input, it ensure no age below 0
+   * and above 200 can be use to filter the search. 
+   * 
+   * @param {interger} age This is the age the user inputs
+   * @param {string} identifier This is the ID of the age 
+   * @returns 
+   * @memberof PatientComponent
+   */
+  handleAgeValidation(age, identifier) {
+      if (age >= 0 && age <= 200 && typeof(age)==="string" && age !== '-0') {
+        this.setState((previousState) => {
+          previousState.ageErrorObject[`${identifier}ErrorMsg`] = '';
+          return previousState;
+        });
+        return true;
+      } else if(age < 0 || age === '-0' || age === -0) {
```

Comment:
Fixed @dkayiwa 
Previously was building on @andela-jomadoye and Paul's issue, but we've fixed this now.